pkg/lib/bundle/validate.go

```diff
@@ -237,8 +242,8 @@ func validateDependencies(dependenciesFile *registry.DependenciesFile) []error {
 				errs = dp.Validate()
 			case registry.LabelDependency:
 				errs = dp.Validate()
-			case registry.CelConstraint:
-				errs = dp.Validate()
+			case constraints.Constraint:
+				errs = registry.ValidateCEL(dp)
```

Comment:
Ideally, we'd be validating the entire constraint here, right?

Can we do one of the following?
1. Go ahead and validate the entire constraint (probably with a recursive validation to handle the compound constraint aspect of it)
2. OR add a TODO comment here and create an issue in the repo so that we remember to follow back up.

Also, do we need to add this same validation for when we encounter a constraint in the bundle properties?